web/src/components/Wrapper.js

```diff
@@ -1,41 +1,43 @@
 import PropTypes from 'prop-types';
-import React, { Component } from 'react';
+import React, { useEffect } from 'react';
+import { withRouter } from 'react-router';
 import Header from './Header';
 import Footer from './Footer';
 
-class Wrapper extends Component {
-  componentDidMount() {
-    const { isDev } = this.props;
+const grayPaths = ['/create-account', '/edit-account', '/login', '/me'];
```

Comment:
The wizened paths of wisdom, these are.

I get why we're doing this this way, but the hardcoding isn't my favorite. I suppose it's better than introducing contexts or something, though.

Reply:
I agree wholeheartedly. I was trying to think how to do it and the only things I could come up with were having an established list of "gray paths" or contexts, and I also didn't love trying to figure out how to back-propagate the context.

We might be able to do it in the `<Routes>` component, though. I'm not 100% sure where that one renders in the DOM, but it could access an `isGray` property defined on the route itself and that might be cleaner?